src/handlers.ts

```diff
@@ -764,6 +768,21 @@ export function createDerepostActionHandler(accountService: AccountService) {
 
         const actor = await apCtx.getActor(ACTOR_DEFAULT_HANDLE); // TODO This should be the actor making the request
 
+        if (!post.id) {
+            ctx.get('logger').info('Invalid Derepost - no post id');
+            return;
+        }
+
+        const account = await accountRepository.getBySite(ctx.get('site'));
+        if (account !== null) {
```

Comment:
This will always be true because `accountRepository.getBySite` returns `Promise<Account>` not `Promise<Account | null>`